packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_panel.dart

```diff
@@ -9,36 +9,41 @@ import 'package:devtools_app_shared/utils.dart';
 import 'package:dtd/dtd.dart';
 import 'package:flutter/material.dart';
 
-import '../../service/editor/editor_client.dart';
-import '../../shared/analytics/analytics.dart' as ga;
-import '../../shared/analytics/constants.dart';
-import '../../shared/ui/common_widgets.dart';
-import '../ide_shared/property_editor/property_editor_sidebar.dart';
+import '../../../service/editor/editor_client.dart';
+import '../../../shared/analytics/analytics.dart' as ga;
+import '../../../shared/analytics/constants.dart';
+import '../../../shared/ui/common_widgets.dart';
+import 'property_editor_controller.dart';
+import 'property_editor_view.dart';
 
 /// The side panel for the Property Editor.
-class PropertyEditorSidebarPanel extends StatefulWidget {
-  const PropertyEditorSidebarPanel(this.dtd, {super.key});
+class PropertyEditorPanel extends StatefulWidget {
+  const PropertyEditorPanel(this.dtd, {super.key});
 
   final DartToolingDaemon dtd;
 
   @override
-  State<PropertyEditorSidebarPanel> createState() =>
-      _PropertyEditorSidebarPanelState();
+  State<PropertyEditorPanel> createState() => _PropertyEditorPanelState();
 }
 
-class _PropertyEditorSidebarPanelState
-    extends State<PropertyEditorSidebarPanel> {
-  _PropertyEditorSidebarPanelState();
+class _PropertyEditorPanelState extends State<PropertyEditorPanel> {
+  _PropertyEditorPanelState();
 
   Future<EditorClient>? _editor;
+  PropertyEditorController? _propertyEditorController;
 
   @override
   void initState() {
     super.initState();
 
     final editor = EditorClient(widget.dtd);
     ga.screen(PropertyEditorEvents.id);
-    unawaited(_editor = editor.initialized.then((_) => editor));
+    unawaited(
+      _editor = editor.initialized.then((_) {
+        _propertyEditorController = PropertyEditorController(editor);
```

Comment:
call dispose on this controller in the dispose method for this state class

Reply:
done!